tanner/dbutils.py

```diff
@@ -142,13 +142,16 @@ def time_convertor(time):
 
                 for path in session["paths"]:
                     timestamp = time_convertor(path["timestamp"])
+                    attackType = AttackType[path["attack_type"]].value
+                    if not attackType:
```

Comment:
how this solves KeyError? if path doesn't have attack_type you will get an exception anyway

```
  File "/usr/local/lib/python3.7/site-packages/Tanner-0.6.0-py3.7.egg/tanner/server.py", line 106, in on_shutdown
  File "/usr/local/lib/python3.7/site-packages/Tanner-0.6.0-py3.7.egg/tanner/sessions/session_manager.py", line 83, in delete_sessions_on_shutdown
  File "/usr/local/lib/python3.7/site-packages/Tanner-0.6.0-py3.7.egg/tanner/sessions/session_manager.py", line 98, in delete_session
  File "/usr/local/lib/python3.7/site-packages/Tanner-0.6.0-py3.7.egg/tanner/sessions/session_analyzer.py", line 32, in analyze
  File "/usr/local/lib/python3.7/site-packages/Tanner-0.6.0-py3.7.egg/tanner/sessions/session_analyzer.py", line 39, in save_session
  File "/usr/local/lib/python3.7/site-packages/Tanner-0.6.0-py3.7.egg/tanner/dbutils.py", line 146, in add_analyzed_data
KeyError: 'attack_type'
```